react/features/base/react/components/web/Text.js

```diff
@@ -14,6 +14,6 @@ export default class Text extends Component {
      * @returns {ReactElement}
      */
     render() {
-        return React.createElement('p', this.props);
+        return React.createElement('span', this.props);
```

Comment:
This looks like a change that affects a lot of existing code. I don't know to what extent, but I would put such things into separate commits. Ideally such commit would explain on why is this necessary. So I'm asking here: why is that change necessary ?

Reply:
The initial layout for the dialog was including 
`<Text>....</Text><Text onPress= {...}>...</Text><Text>...</Text>` and all this was supposed to be displayed without new lines which is impossible with `p`. I did the change and tested/fixed the other places where `Text` was used. 

After that we switched to the current layout where this thing is not needed anymore but I decided to keep the change because `span` behaves more like the react-native's `Text` component and we may have problems in future with it.

If you insist I can remove this or extract in another commit or even extract in another PR. Let me know if you want me to do any changes.